testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -5420,8 +5441,8 @@ public void subTestIntervalYearPositive() {
    */
   public void subTestIntervalYearToMonthPositive() {
     // default precision
-    expr("interval '1-2' year to month")
-        .ok("INTERVAL '1-2' YEAR TO MONTH");
+    expr("interval '1:2' year to month")
+        .ok("INTERVAL '1:2' YEAR TO MONTH");
```

Comment:
Did you intend to change the syntax from hyphen to colon? I believe hyphen was correct already.

Reply:
Oh. This is a typo. Thanks for catching that.